app/views/shared/_reply_form.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= form_with(model: [micropost, comment], local: true, html: { class: "form-post", id: "form-post-#{micropost.id}", remote: true }) do |f| %> 
+    <%= render 'shared/error_messages', object: f.object %> 
+    <div class="field">
+        <%= f.text_field :content, placeholder: "Comment...", class: "box-input box-input-cmt", id: "#{micropost.id}" %> 
+    </div>
+    <%= f.hidden_field :micropost_id, value: micropost.id, class: "micropost-id-cmt" %>
+    <%= f.hidden_field :user_id, value: current_user.id %>
+    <%= f.submit "Comment", class: "btn btn-primary btn-cmt btn-cmt-#{micropost.id}" %> 
+    <span class="image">
+        <%= f.file_field :image, accept: "image/jpeg,image/gif,image/png" %> 
+    </span>
+<% end %>
+<%= javascript_include_tag "exception_img", "data-track-turbo": "true", defer: true %>
+<%= javascript_include_tag "form_post_reply", "data-track-turbo": "true", defer: true %>
```

Comment:
This script is rendered every time there is a new comment. This makes the DOM very heavy.
There is a rule to remember, never write scripts or embed scripts in the DOM.